### DISCUSSION TITLE
Max number of USBMIDIDevices on RPI3

### DISCUSSION THREAD

**Discussion by liv003** [probonopd/MiniDexed#375] [Q&A]
Hello,
It's possible to connect 3 USBmidi device on RPI3?

In config.h i find :

```
#if RASPPI <= 3
	static const unsigned MaxUSBMIDIDevices = 2;
#else
	static const unsigned MaxUSBMIDIDevices = 4;
#endif
```

Why is there a limit to only 2 devices for rpi3?

**Comment by probonopd** [comment]
You could change that number and see whether everything still works for you. @rsta2 wrote in https://github.com/probonopd/MiniDexed/pull/37:

> Support multiple USB MIDI inputs at once (two on Raspberry Pi 1-3, four on Raspberry Pi 4).
